lib/screens/drop_in_checkout/checkout/checkout.dart

```diff
@@ -6,7 +6,7 @@ import 'package:flutter/foundation.dart';
 import 'package:http/http.dart' as http;
 import 'package:inai_flutter_sdk/inai_flutter_sdk.dart';
 import 'package:shared_preferences/shared_preferences.dart';
-import '../../contants.dart';
+import '../../../contants.dart';
```

Comment:
have we changed the folder structure ?

Reply:
Yes...Added a seperate folder for checkout